src/EventsPage/components/AllUpcomingEvents.js

```diff
@@ -6,6 +6,22 @@ import _ from 'lodash';
 
 const sortByDate = (events = []) => (_.orderBy(events, 'date', 'desc'))
 
+var today = new Date();
+var dd = today.getDate();
+var mm = today.getMonth()+1; //January is 0!
+var yyyy = today.getFullYear();
+
+if(dd<10) {
+    dd = '0'+dd
+} 
+
+if(mm<10) {
+    mm = '0'+mm
+} 
+
+today = mm + '/' + dd + '/' + yyyy;
```

Comment:
I think this whole thing could be done with:
```
let today = new Date();
today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit' });
```

Also if we're just using this to cheese the date comparison on line 47, maybe we should move that code into the `renderEventList` function?